### ISSUE TITLE
Hierarchy roles with rights inheritance

### ISSUE THREAD

**Issue by MaxSor** [allcount/allcountjs#18]

**Comment by paveltiunov** [comment]
Max, I think need to add some clarification on how it can be configured.
Let me try:
```
A.app({
  roles: function (Roles) { 
    return ['foo', 'bar', Roles.extend('fooAndBar', ['foo', 'bar'])] 
  }
})
```
Would it be sufficient?

**Comment by MaxSor** [comment]
Yes, it seems good. But after while i think this feature isn't so necessary now. May be we could find some other ways to organize roles and permissions later. User groups for example.
I'm decide to close the issue for now.
